executor_bot/src/dex/openbook.ts

```diff
@@ -0,0 +1,141 @@
+import { Market, OpenOrders } from "@project-serum/serum";
+import { TOKEN_PROGRAM_ID } from "@raydium-io/raydium-sdk";
+import {
+  PublicKey,
+  AccountMeta,
+  SYSVAR_RENT_PUBKEY,
+  Connection,
+  Transaction,
+  Keypair,
+  sendAndConfirmTransaction,
+} from "@solana/web3.js";
+import BN from "bn.js";
+
+export const OPENBOOK_V3_PROGRAM_ID = new PublicKey(
+  "srmqPvymJeFKQ4zGQed1GFppgkRHL9kaELCbyksJtPX"
+);
+
+export const openBookTradeAccounts = async (
```

Comment:
This looks like an exact replica of `@mithraic-labs/poseidon` `OpenBookDex.tradeAccounts`. Lets DRY it up